roles/openstack/neutron/tasks/cleanup.yml

```diff
@@ -0,0 +1,26 @@
+---
+# This file is used to cleanup OpenStack neutron resources
+
+- name: Cleanup routers
```

Comment:
All these arguments are wrong. you need to use `"{{ arg }}"`  and not `{{ arg }}`.